### PR TITLE
feat: Command Description

### LINKED ISSUE #244: Add a box to linutil for a description of the program.
Otherwise, you could place a short description of the program beside the name of the program when hovered. This would be great for new linux users who have no idea what a "kitty" or a "alacritty" is.

### PR DESCRIPTION
**Pull request by jeevithakannan2**
# Pull Request

## Title
Implement command description for commands just like how command preview works

## Type of Change
- [x] New feature

## Description
Add command description be using **D** key just like how command preview works

about

## Testing
Works as expected.

## Impact
When new scripts are added a description must be added in the toml for command description to work

## Issue related to PR
- Resolves #244

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: tui/src/state.rs [hunk 1]
```diff
@@ -234,6 +234,7 @@ impl AppState {
                 KeyCode::Char('j') | KeyCode::Down => self.selection.select_next(),
                 KeyCode::Char('k') | KeyCode::Up => self.selection.select_previous(),
                 KeyCode::Char('p') => self.enable_preview(),
+                KeyCode::Char('a') => self.enable_about(),
```
Comment (nnyyxxxx) [inline tui/src/state.rs:237]:
```suggestion
                KeyCode::Char('d') => self.enable_about(),
```

File: tui/src/float.rs [hunk 2]
```diff
@@ -64,7 +64,9 @@ impl Float {
     // Returns true if the floating window is finished.
     pub fn handle_key_event(&mut self, key: &KeyEvent) -> bool {
         match key.code {
-            KeyCode::Enter | KeyCode::Char('p') | KeyCode::Esc if self.content.is_finished() => {
+            KeyCode::Enter | KeyCode::Char('p') | KeyCode::Char('a') | KeyCode::Esc
```
Comment (nnyyxxxx) [inline tui/src/float.rs:67]:
```suggestion
            KeyCode::Enter | KeyCode::Char('p') | KeyCode::Char('d') | KeyCode::Esc
```

**Comment by nnyyxxxx** [comment]
Not sure why the keybind is "a" when "d" is the first letter of "description"

**Review by nnyyxxxx — APPROVED** [review]
Awesome

**Comment by nnyyxxxx** [comment]
> > Not sure why the keybind is "a" when "d" is the first letter of "description"
> 
> It was initially thought to the function name as about page and the key **A** for about but command description suited it better so I refactored the function names to description with **D** as keybind

yeah thanks

**Review by adamperkowski — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Comment by ChrisTitusTech** [comment]
Now this is PERFECT! Thank you @jeevithakannan2 exactly what I was thinking.

**Pull request merged**